### PR TITLE
Server creation procedure #4

### LINKED ISSUE #4: Server: create minecraft server
- Write a service, that creates the minecraft server from a jar
- Keep it generic!
- Use Process Class

_Keep in mind, that you need to run the file two times and accept the EULA to fully create the server_

### PR DESCRIPTION
**Pull request by giaenuuu**

### CODE REVIEW

File: server/Domain/Serverproperties.cs [hunk 1]
```diff
@@ -1,119 +1,176 @@
...
 public class ServerProperties
 {
-    public int spawnProtection {get; set;}
+    [JsonPropertyName("spawn_protection")]
+    public int spawnProtection { get; set; }
 
-    public int maxTickTime {get; set;}
-    
-    public bool forceGamemode {get; set;}
+    [JsonPropertyName("max-tick-tim")]
```
Comment (fl3xm3ist3r) [inline server/Domain/Serverproperties.cs]: typo

Comment (fl3xm3ist3r) [inline server/Domain/Serverproperties.cs]: typo

File: server/Domain/Serverproperties.cs [hunk 2]
```diff
@@ -1,119 +1,176 @@
...
-    public bool useNativeTransport {get; set;}
+    [JsonPropertyName("level-seed")]
+    public string? levelSeed { get; set; }
 
-    public string motd {get; set;}
+    [JsonPropertyName("use-native-transport")]
+    public bool useNativeTransport { get; set; }
 
-    public bool enableRcon {get; set;}
+    [JsonPropertyName("modt")]
```
Comment (fl3xm3ist3r) [inline server/Domain/Serverproperties.cs]: typo?

Comment (fl3xm3ist3r) [reply server/Domain/Serverproperties.cs]: modt or motd?

File: server/Service/InputModels/ServerPropertiesInputModel.cs
Comment (fl3xm3ist3r) [inline server/Service/InputModels/ServerPropertiesInputModel.cs:1]: adjust to correctements above

**Review by fl3xm3ist3r — DISMISSED** [review]

File: server/Domain/Player.cs [hunk 3]
```diff
@@ -1,8 +1,8 @@
-﻿namespace domain;
+﻿namespace Domain;
 
 public class Player : NamedEntityBase
 {
-    public Player(string name, Guid id) : base(string name) {
```
Comment (kubaw161) [inline server/Domain/Player.cs:5]: wieso ohni type?

File: server/Service/InputModels/ServerPropertiesInputModel.cs [hunk 4]
```diff
@@ -0,0 +1,175 @@
+﻿using Domain.Enums;
+using CrossCutting.Attributes;
+
+namespace Service.InputModels;
+
+public class ServerPropertiesInputModel
+{
+    [ServerPropertyName("spawn_protection")]
```
Comment (kubaw161) [inline server/Service/InputModels/ServerPropertiesInputModel.cs]: nid "spawn-protection" ?

**Review by hofmann-andreas — APPROVED** [review]

**Pull request merged**